app/controllers/catalog_controller.rb

```diff
@@ -1304,9 +1304,10 @@ def set_form_vars
     @edit[:new][:provision_cost] = @record.provision_cost
     @edit[:new][:display]  = @record.display ? @record.display : false
     @edit[:new][:catalog_id] = @record.service_template_catalog ? @record.service_template_catalog.id : nil
-    @edit[:new][:available_catalogs] = ServiceTemplateCatalog.all
-                                       .collect { |stc| [stc.name, stc.id] }
-                                       .sort
+    @edit[:new][:available_catalogs] = ServiceTemplateCatalog.all.collect do |stc|
```

Comment:
this place is in UI here:
![service-template-catalogs](https://cloud.githubusercontent.com/assets/14937244/14857298/0f4b8c74-0c9c-11e6-9f8a-36d2bc288bba.jpg)

`ServiceTemplateCatalog` model has tenant's ancestor strategy (https://github.com/ManageIQ/manageiq/blob/master/app/models/rbac.rb#L72) and we are using it on some places already.

So I am suggesting to use RBAC here, something like this:  
`@edit[:new][:available_catalogs] = rbac_filtered_objects(ServiceTemplateCatalog.all).collect do |stc|`

do you agree @gtanzillo ?